src/components/Header/MobileHeaderMenu.tsx

```diff
@@ -1,36 +1,39 @@
+import { useAppStore } from "@/hooks/useAppStore";
```

Comment:
Same comments regarding the menu as above.

Reply:
Yep, the toggle is completely removed from the header now